portfolio/src/main/java/com/google/sps/servlets/BlogCommentServlet.java

```diff
@@ -152,5 +162,53 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
       return imagesService.getServingUrl(options);
     }
   }
+
+  /* returns the sentiment score of the message */
+  private float getSentimentScore(String message) {
+    
+    try {
+      Document doc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+      LanguageServiceClient languageService = LanguageServiceClient.create();
+      
+      Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+      float score = sentiment.getScore();
+      languageService.close();
+      return score;
+    }
+    catch(IOException e) {
+      e.printStackTrace();
+    }
+    return 0;
+  }
+
+  /* returns the content classifications of the message */
+  private String classifyContent(String message) {
```

Comment:
Same for content classification.

Reply:
Same here. I think you just forgot to delete this method after moving it.